packages/@n8n/api-types/src/dto/auth/__tests__/login-request.dto.test.ts

```diff
@@ -30,7 +30,7 @@ describe('LoginRequestDto', () => {
 			{
 				name: 'login request with only mfaRecoveryCode',
 				request: {
-					email: 'test@example.com',
+					emailOrLdapUsername: 'test@example.com',
 					password: 'securePassword123',
 					mfaRecoveryCode: 'recovery-code-123',
 				},
```

Comment:
Do we have some tests with `emailOrLdapUsername` as a valid username and not an email?
Should we add some if not?

Reply:
We do not what the "username" is going to be. Could be a username (string), a email or a number, or a combination of characters including number.